core/app/components/DataTable/PubsDataTable/columns.tsx

```diff
@@ -0,0 +1,73 @@
+"use client";
+
+import type { ColumnDef } from "@tanstack/react-table";
+
+import * as React from "react";
+
+import type { ProcessedPub } from "contracts";
+import type { DataTableRowAction } from "ui/data-table-paged";
+import { Checkbox } from "ui/checkbox";
+import { DataTableColumnHeader } from "ui/data-table";
+
+import { dateFormatOptions } from "~/lib/dates";
+import { getPubTitle } from "~/lib/pubs";
+
+interface GetColumnsProps {
+	setRowAction?: React.Dispatch<
+		React.SetStateAction<DataTableRowAction<ProcessedPub<{ withPubType: true }>> | null>
+	>;
+}
+
+export function getColumns({
```

Comment:
I think once we use this table more, we should pass different columns in. but here's an example of the simple columns we need for adding related pubs (just the name and `updatedAt` and checkboxes)